collections/Hamilton/MS_Hamilton_49.xml

```diff
@@ -50,16 +43,41 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite: two parts</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="mixed">
```

Comment:
I don't think it is helpful to use "mixed" in this context - I think it should be reserved for codicological units that originally combine the two materials not for cases where they are only in the same codex by later happenstance

Reply:
Would you simply delete the overarching material statement in such cases?

Reply:
Yes